libminifi/test/opencv-tests/CMakeLists.txt

```diff
@@ -24,14 +24,14 @@ SET(OPENCV_TEST_COUNT 0)
 FOREACH(testfile ${OPENCV_TESTS})
 	get_filename_component(testfilename "${testfile}" NAME_WE)
 	add_executable("${testfilename}" "${testfile}")
-	createTests("${testfilename}")
 	target_include_directories(${testfilename} PRIVATE BEFORE "${CMAKE_SOURCE_DIR}/extensions/opencv/")
-
+	target_include_directories(${testfilename} PRIVATE BEFORE "${CMAKE_SOURCE_DIR}/extensions/standard-processors")
 	if (APPLE)
-		target_link_libraries (${testfilename} -Wl,-all_load minifi-opencv)
+		target_link_libraries (${testfilename} -Wl,-all_load minifi-opencv minifi-standard-processors)
 	else ()
-		target_link_libraries (${testfilename} -Wl,--whole-archive minifi-opencv -Wl,--no-whole-archive)
```

Comment:
Could you to this the "standard" way we do this? (Just setting variables in conditions to match platform and have only one linker command.